### DISCUSSION TITLE
Possible Auto mine function similar to auto fish?

### DISCUSSION THREAD

**Discussion by Zippedy** [ORelio/Minecraft-Console-Client#1789] [Ideas]
I was curious if there'd be a way to implement an Auto Mine. I was thinking that you would have to log in the character, point it where you want it to mine, put pickaxes in it's hot bar, and just let it rip. The server I play on allows afk mining on its SkyBlock.

I know that there is the inventory handling thing, so would giving it a full inventory of pickaxes and letting it mine be an issue?

There could be a way that as soon as the pickaxe is about to die, lets say 30 uses left, it switches to the next.

My computer can handle the resources that the client can draw, and I actively host 120 alts as it is.

This was just a thought in the back of my mind. I can go in depth on what I'd like it to have, and I can donate/subscribe to get this added.

Please feel free to drop a comment and add your thoughts!

**Comment by ReinforceZwei** [comment]
Perhaps this [MineCube.cs](https://github.com/ORelio/Minecraft-Console-Client/blob/master/MinecraftClient/config/ChatBots/MineCube.cs) script could help you.

**Reply by Zippedy** [reply]
> Perhaps this [MineCube.cs](https://github.com/ORelio/Minecraft-Console-Client/blob/master/MinecraftClient/config/ChatBots/MineCube.cs) script could help you.

I will give this a shot. Thanks for your help!

**Reply by Zippedy** [reply]
How do you implement it? I can't seem to figure out how to add it

**Reply by Zippedy** [reply]
Disregard. I found out how. :P My brain thinks its Monday, and it's Saturday.... Hahaha.

**Comment by Daenges** [comment]
There is also the [CobblestoneMiner.cs](https://github.com/MCCTeam/Minecraft-Console-Client/blob/master/MinecraftClient/config/ChatBots/CobblestoneMiner.cs) script in the same folder. I don't know whether this is more efficient with a normal cobblestone generator, since MineCube.cs is rather for excavating bigger holes.
